apps/andi/lib/andi/input_schemas/datasets/dataset.ex

```diff
@@ -34,6 +36,14 @@ defmodule Andi.InputSchemas.Datasets.Dataset do
     |> cast_assoc(:business, with: &Business.changeset/2)
   end
 
+  def changeset_for_draft(dataset, %{owner: owner} = changes) do
```

Comment:
Not in love with having two heads for this, but I can't see a way around having one way put an owner in and the other not.